src/features/wallet/SideBarMenu.tsx

```diff
@@ -110,11 +110,11 @@ export function SideBarMenu({
             <Image src={CollapseIcon} width={15} height={24} alt="" />
           </button>
         )}
-        <div className="w-full h-full">
-          <div className="w-full bg-blue-500 py-2 px-3.5 text-white tracking-wide rounded-t-sm">
+        <div className="w-full h-full flex flex-col overflow-y-auto">
```

Comment:
Changes in this file are form a commit I just from Danylo a few minutes ago. I asked him yesterday to improve the scrolling behavior of the sidebar menu. The bottom was getting cut off